modules/500-upmeter/hooks/clean_probe_garbage.go

```diff
@@ -58,7 +59,8 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 
 		for _, r := range repos {
 			if err := cleanGarbage(ctx, r); err != nil {
-				return err
+				// The queue shouldn't be stopped event if there is an API error
```

Comment:
Maybe it's better also to move all upmeter schedules to separate queue?

Reply:
I find it a good idea.

Reply:
Done